components/images/README.md

```diff
@@ -0,0 +1 @@
+This is Pattern Lab's default images directory. We recommend putting theme images into `/images` or into a relevant component directory.
```

Comment:
Let's specify that they're only for pattern lab. Something like:

> This is Pattern Lab's default images directory. Images that only apply to the Pattern Lab instance should be placed here. e.g. One image of each dimension (portrait, landscape, square, etc.) to demonstrate the various components with images.
> 
> We recommend putting theme images into the theme's root-level `/images`directory or into a relevant component directory.

Reply:
I like this, updated.